### ISSUE TITLE
USER ISSUE: Plant Already Exists

### ISSUE THREAD

**Issue by werkrat** [StrangeLoopGames/EcoIssues#6784]
**Version:** 0.7.3.1 beta

**Steps to Reproduce:**
Some, not all, spots on the ground register as something is already planted, so attempting to plant something fails. Even if you dig up the dirt, drop it, till it, try to plant, it still fails.

**Expected behavior:**
Seed should plant

**Actual behavior:**
Fails

**Comment by GophTheGreat** [comment]
+1 from the support queue:

"I'm running into an issue where I can't replant crops in certain areas because the Plant already exists."

Told them to comment here if they have any more usable info.

**Comment by seanimusprime** [comment]
I was the one who contacted via email.
Version: Latest

Running a server to play with 3 other friends.
I have tried the following things to remedy it, and to no avail:
1. Digging and filling it back in with dirt.
2. Digging and filling back in with dirt, stacking a Hewn Wood block on top, and then removing the Hewn Wood.
3. Digging out the sections and filling in with Hewn Wood blocks.

I have also tried various combinations of the following steps along with logging off the server, restarting the game and/or restarting the game. No combination of any of these resulted in a resolution of the bug.

In addition to this, we are running several modifications. However this issue existed prior to us using the mods, so I don't feel as if there is a correlation, especially since the problem continues to exist after clearing and restoring the default mods folder.
